src/renderer/modules/components/index.ts

```diff
@@ -91,6 +91,11 @@ export type { TextInputType };
 export let TextInput: TextInputType;
 importTimeout("Input", import("./TextInput"), (mod) => (TextInput = mod.default));
 
+import type { SliderType } from "./Slider";
+export type { SliderType };
+export let Slider: SliderType;
+importTimeout("Input", import("./Slider"), (mod) => (Slider = mod.default));
```

Comment:
Shouldn't "Input" be changed to "Slider"?

Reply:
Fixed